Convert markdown documentation to asciidoc

SaschaWillems opened the pull request:
## Description

This PR converts all markdown documentation files of our repo to asciidoc (third_party modules etc. are excluded). There are no changes to the general structure or contents, so this is mostly a 1:1 to change that most users won't notice. The PR does fix a few broken links from master, does some minor markup and formatting fixes and changes some image links to be better suited for the Antora documentation project. There are only slight visual changes due to differences in the github asciidoc renderer.

This conversion is a base for getting the samples into the Vulkan documentation site.

Fixes #693

## General Checklist:

Please ensure the following points are checked:

- [ ] My code follows the [coding style](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Code-Style)
- [ ] I have reviewed file [licenses](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Copyright-Notice-and-License-Template)
- [ ] I have commented any added functions (in line with Doxygen)
- [ ] I have commented any code that could be hard to understand
- [ ] My changes do not add any new compiler warnings
- [ ] My changes do not add any new validation layer errors or warnings
- [ ] I have used existing framework/helper functions where possible
- [ ] My changes do not add any regressions
- [ ] I have tested every sample to ensure everything runs correctly
- [ ] This PR describes the scope and expected impact of the changes I am making

 Note: The Samples CI runs a number of checks including:
 - [ ] I have updated the header Copyright to reflect the current year (CI build will fail if Copyright is out of date)
 - [ ] My changes build on Windows, Linux, macOS and Android. Otherwise I have [documented any exceptions](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#General-Requirements)

SaschaWillems committed:
Converted markdown to asciidoc

SaschaWillems committed:
Corrected links from .md to .adoc

SaschaWillems committed:
Remove conversion script

SaschaWillems committed:
Replaced markdown in sample template with asciidoc

SaschaWillems committed:
Fix link to sample creation

Already broken on master (markdown)

SaschaWillems commented:
Note: This will break with copyright checks, as a lot of files have changed. I don't want to change copyright in all these files, so if we decide to merge we should just ignore failing copyright checks.

SaschaWillems committed:
Fixed broken link to AMD presentation

SaschaWillems committed:
Revert conversion of pull request template from markdown to asciidoc

Github requires it to be in md

SaschaWillems committed:
Fix asciidoc formatting issues

SaschaWillems committed:
Fix asciidoc formatting issues

SaschaWillems committed:
Fixed link

SaschaWillems committed:
Fixed copyrights

SaschaWillems committed:
Merge branch 'main' into asciidoc_conversion

SaschaWillems committed:
Applied latest markdown changes to asciidoc files

gpx1000 reviewed (dismissed):
Thanks for the work on this Sascha!

tomadamatkinson reviewed (dismissed):
LGTM! Thanks for pushing the documentation goal forward :fire:

asuessenbach commented on CONTRIBUTING.adoc:
```diff
@@ -0,0 +1,152 @@
+A sample must consistently apply a single code style.
+
+A `.clang-format` file is included with this repository, please use `clang-format -style=file` to verify the code style.
+Ensure that you are using clang-format 8.
```
Is clang-format 8 still the correct version?

SaschaWillems replied:
Not sure. This is just a conversion (and some minor fixes) of the current documentation. If this is outdated it would be best to fix it with the current markdown and I'll merge that change.

tomadamatkinson replied:
15 is the correct version. We should do a documentation pass separate to this PR

SaschaWillems committed:
Adjust image links

This doesn't change anything in how the readmes are displayed in github, but it makes antora documentation generation easier

SaschaWillems committed:
Adjust image links

This doesn't change anything in how the readmes are displayed in github, but it makes antora documentation generation easier

SaschaWillems committed:
Adjust image links

This doesn't change anything in how the readmes are displayed in github, but it makes antora documentation generation easier

SaschaWillems committed:
Adjust image links

This doesn't change anything in how the readmes are displayed in github, but it makes antora documentation generation easier

SaschaWillems committed:
Adjust image links

This doesn't change anything in how the readmes are displayed in github, but it makes antora documentation generation easier

SaschaWillems committed:
Fixed images

SaschaWillems committed:
Fix link to proposal
